src/test/java/uk/co/ramp/covid/simulation/population/PopulationTest.java

```diff
@@ -122,6 +123,7 @@ public void populateHouseholds() {
         }
     }
 
+    @Ignore("Fails with some RNG seeds.")
     @Test (expected = ImpossibleAllocationException.class )
```

Comment:
I would expect this test to pass (throw an exception) regardless of the random number seed

Reply:
Could you explain why it should throw an exception?  (I didn't understand the test.)

Reply:
The test isn't particularly robust to RNG and I agree with bob we might be better just keeping the next one (where we have an explicit "too many households"). Usually allocation needs < 1 household per person (since some require 2 adults etc), but sometimes you get lucky and can allocate 1 person per household which is why this sometimes succeeds when it should fail

I suggest we drop this.